wallet/invoices.c

```diff
@@ -96,10 +97,22 @@ static struct invoice_details *wallet_stmt2invoice_details(const tal_t *ctx,
 		dtl->pay_index = db_col_u64(stmt, "pay_index");
 		db_col_amount_msat(stmt, "msatoshi_received", &dtl->received);
 		dtl->paid_timestamp = db_col_u64(stmt, "paid_timestamp");
+		if (!db_col_is_null(stmt, "paid_txid")) {
+			paid_outpoint = tal(ctx, struct bitcoin_outpoint);
+			db_col_txid(stmt, "paid_txid",
+					&paid_outpoint->txid);
+			paid_outpoint->n
+				= db_col_int(stmt, "paid_outnum");
+			dtl->paid_outpoint = paid_outpoint;
+		} else {
+			dtl->paid_outpoint = NULL;
```

Comment:
db_col_ignore(stmt, "paid_outnum");

Reply:
You don't need to ignore paid_txid, you tested it...

Reply:
Ok, thanks!